exercises/palindrome-products/example.py

```diff
@@ -1,53 +1,51 @@
+
 from __future__ import division
 from itertools import chain
 from math import log10, floor, ceil
 
 
-def largest_palindrome(max_factor, min_factor):
-    return get_extreme_palindrome_with_factors(max_factor, min_factor,
-                                               "largest")
+def largest(min, max):
```

Comment:
`min` and `max` are reserved.

```suggestion
def largest(min_factor, max_factor):
```